ynput/OpenPype
# Nuke, DL: Returning Suspended Publishing attribute

## Changelog Description
Old Nuke Publisher's feature for suspended publishing job on render farm was added back to the current Publisher.

## Additional info
This is how it suppose to look in publisher:
![image](https://user-images.githubusercontent.com/40640033/227904656-58f462d3-608e-46ea-866a-815b17f1ce3a.png)


## Testing notes for Nuke:
1. Open Nuke in favorite testing workfile
2. Open publisher on render instance
3. Set the instance to Farm render
4. Set `Suspended publish` (img) as activated
5. Publish to farm (start your webservice)
6. Notice that python job is set to `suspended`
